Pipfile

```diff
@@ -12,3 +12,8 @@ coverage = "*"
 coveralls = "*"
 isort = "*"
 flake8 = "*"
+
+[scripts]
+lint = "python -m flake8 --max-line-length=110 --exclude=migrations,./tests/*settings*.py"
+isort = "python -m isort --check-only --recursive --diff"
```

Comment:
can we move this logic to the [setup.cfg as we have done in django-freeradius](https://github.com/openwisp/django-freeradius/blob/master/setup.cfg)? That way the flake8 and isort default options are used also without pipenv.

Reply:
Yeah, that's a good idea.

Reply:
I've addressed this in the most recent commit.